pallets/migrations/src/tests.rs

```diff
@@ -265,3 +265,47 @@ fn overweight_migrations_tolerated() {
 		},
 	);
 }
+
+#[cfg(all(test, feature = "try-runtime"))]
```

Comment:
By making this only run with `cargo test --features=try-runtime`, this test case is guaranteed to rot.

We could change CI to either (1) always run tests with `--features=try-runtime` or (2) run a separate pass. Neither seem like great solutions to me. Suggestions welcome.

Reply:
@librelois